zazu/repo/commands.py

```diff
@@ -81,12 +89,93 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('-i', 'interactive', is_flag=True, help='starts interactive repo init')
 @click.pass_context
-def init(ctx):
-    """Initialize repo directory structure"""
-    raise NotImplementedError
-
+def init(ctx, interactive):
+    """Initialize repo directory structure
+    TODo:
+    -if not git repo offer to make one
+    -if git repo, assume user wants current repo to use zazu
+    """
+    
+    def _get_plugin_list(plugin_subclass):
+        """helper function to pull lists of plugins"""
+        plugins = straight.plugin.load('zazu.plugins', subclasses=plugin_subclass)
+        known_types = {p.type().lower(): p.from_config for p in plugins}
+        return known_types.keys()
+
+    def _build_inquiry(name, message, choices=[], checkbox=False):
+        """helper function to build an inquiry"""
+        if not checkbox:
+            inquiry = [
+                inquirer.List(str(name),
+                             message = str(message),
+                             choices = trackers,
+                             ),
+            ]
+
+        else:
+            inquiry = [
+                inquirer.Checkbox(str(name),
+                             message = str(message),
+                             choices = choices,
+                             ),
+            ]
+        
+        return inquirer.prompt(inquiry)
+
+    def _zazu_yaml(issue_tracker={}, stylers={}):
+        """builds zazu.yml file"""
+        zazu_yaml_obj = {}
+        zazu_yaml_obj['issueTracker'] = issue_tracker
+        if stylers:
+            zazu_yaml_obj['style'] = stylers['style']
+        yaml.dump(zazu_yaml_obj, file('zazu.yaml','w'), default_flow_style=False)
+
+    def _getZazuYaml():
+        """gets zazu's yaml"""
+        import pdb
+        pdb.set_trace()
+        zazu_yaml = requests.get('https://raw.githubusercontent.com/stopthatcow/zazu/develop/zazu.yaml')
+
+    # check for git repo in cwd
+    if not os.path.isdir('.git'):
+        event_time = time.gmtime()
+        default_name = time.mktime(event_time)
+
+        # click.prompt does not play well with py format
+        repo_name = click.prompt('No existing git repo found, Name your new repo:', default='zazuRepoCreated_'+str(default_name))
 
+        try:
+            os.mkdir(repo_name)
+            bare_repo = git.Repo.init('{}/{}/.'.format(repo_name, '.git'),bare=True)
```

Comment:
I don't think you need the .git. I think `git.Repo.init(repo_name, bare=True)` will work.